packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.controller.js

```diff
@@ -50,6 +51,16 @@ export default class KubernetesServiceCtrl {
     this.messages = this.messageHandler.getMessages();
   }
 
+  loadPrivateNetowrkDetails() {
+    this.loadingPrivateNetwork = true;
+    return this.Kubernetes.loadPrivateNetowrkDetails(
+      this.projectId,
+      this.cluster,
+    ).finally(() => {
```

Comment:
Nothing is set to store the value, legit ?

Reply:
no, it adds into the cluster object

Reply:
I don't think the `loadPrivateNetowrkDetails ` should update its parameter. It should return the privateNetworkDetails and then in the controller or in the `then` hook we decide to aggregate data with the cluster one. 
Let's say I call this function elsewhere with the same cluster, some unexpected data is added and it could have some side effects

Reply:
I have changed it